README.md

```diff
@@ -161,6 +161,18 @@ Available files, and their formats:
    * JSON
    * Read ID to Kraken output and cleaned read
 
+* `alignments/`: Output, data that will later back the dashboard.
```

Comment:
Can we describe what this stage does? Eg "Output, uses Bowtie2 to align reads found in `hvreads/`"

Reply:
Good point!  Added in 1d2ae18